client/gutenberg/extensions/slideshow/settings.js

```diff
@@ -0,0 +1,64 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+export const settings = {
+	name: 'slideshow',
+	prefix: 'jetpack',
+	title: __( 'Slideshow' ),
+	category: 'jetpack',
+	keywords: [ __( 'image' ) ],
+	description: __( 'Add an interactive slideshow.' ),
+	attributes: {
+		align: {
+			type: 'string',
+		},
+		images: {
+			type: 'array',
+			default: [],
+			source: 'query',
+			selector: '.swiper-slide',
+			query: {
+				alt: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'alt',
+					default: '',
+				},
+				caption: {
+					type: 'string',
+					source: 'html',
+					selector: 'figcaption',
+				},
+				id: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'data-id',
+				},
+				url: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'src',
+				},
+			},
+		},
+		effect: {
+			type: 'string',
+			default: 'slide',
+		},
+	},
+	effectOptions: [
+		{ label: __( 'Slide' ), value: 'slide' },
+		{ label: __( 'Fade' ), value: 'fade' },
+		{ label: __( 'Cover Flow' ), value: 'coverflow' },
+		{ label: __( 'Flip' ), value: 'flip' },
```

Comment:
I wonder if these terms are obvious enough for translators or do they need context?

`_x( 'Fade', 'Slideshow block transition effect' )`